addons/calendar/models/calendar.py

```diff
@@ -445,7 +447,7 @@ def do_mail_reminder(self, alert):
 
         result = False
         if alarm.type == 'email':
-            result = meeting.attendee_ids.filtered(lambda r: r.state != 'declined')._send_mail_to_attendees('calendar.calendar_template_meeting_reminder', force_send=True)
```

Comment:
Actually, thinking about it, I would only modify this call. The reminder was the really problematic use case, so we could limit the modification here. If an issue arise with the other calls, we can change them later.

Reply:
Ok! I removed the modifications for the others cases